### ISSUE TITLE
Unexpected behaviours with "Expand Selection"

### ISSUE THREAD

**Issue by scarlett-hill** [microsoft/pylance-release#3301]
Type: <b>Bug</b>

# Forewords

Now let me first say that I absolutely **love** this extension. Everything is so cool about developing in python with it. I've moved from pycharm to using vscode as my main IDE for a month now. But I've had a few issues with "expand selection".

# Behaviour
### Expected
1. Using "Expand selection" on a multiple line expression should include opening and closing parentheses.
2. Using "Expand selection" inside a function should successively select the arguments, then the signature, then the whole line

### Actual
1. The selection doesn't enclose the last parenthesis
2. The selection directly selects the whole line after the current argument.

### Example for unexpected behaviour 1
![Animation](https://user-images.githubusercontent.com/39465428/188963512-d556847f-af1a-4961-8ca8-b95036f3f7f8.gif)

### Example for unexpected behaviour 2
![Animation2](https://user-images.githubusercontent.com/39465428/188964879-508d6ed0-282b-4738-b967-8de425cf9335.gif)

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.7
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Poetry
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

formatting
• provider: "black"

testing
• pytestArgs: "<placeholder>"
• pytestEnabled: true

```

</p>
</details>

Extension version: 2022.14.0
VS Code version: Code 1.71.0 (784b0177c56c607789f9638da7b6bf3230d47a8c, 2022-09-01T07:36:10.600Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

**Comment by bschnurr** [comment]
can't repo the first case
```python
foo = 1

y =(
    foo
)
```

**Comment by bschnurr** [comment]
thanks..
will update to match typescript behaviour to select all arguments before selecting the entire call.

**Comment by scarlett-hill** [comment]
> can't repo the first case
> 
> ```python
> foo = 1
> 
> y =(
>     foo
> )
> ```

Alright, I confirm that the first case has been fixed for some reason.

**Comment by scarlett-hill** [comment]
> thanks.. will update to match typescript behaviour to select all arguments before selecting the entire call.

Thanks!

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2022.10.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20221031-19-October-2022-PreRelease)
